src/system_library.jl

```diff
@@ -1,11 +1,18 @@
-include(joinpath(DATA_DIR, "psy_data", "generation_cost_function_data.jl"))
-include(joinpath(DATA_DIR, "psy_data", "data_5bus_pu.jl"))
-include(joinpath(DATA_DIR, "psy_data", "data_10bus_ac_dc_pu.jl"))
-include(joinpath(DATA_DIR, "psy_data", "data_14bus_pu.jl"))
-include(joinpath(DATA_DIR, "psid_tests", "data_tests", "dynamic_test_data.jl"))
-include(joinpath(DATA_DIR, "psid_tests", "data_examples", "load_tutorial_functions.jl"))
+include(joinpath(get_pstd_data_dir(), "psy_data", "generation_cost_function_data.jl"))
```

Comment:
This may be problematic. It at least means we won't easily be able to change the data dir without restarting the REPL. Will this also get baked in at compilation time? Will have to test.

Reply:
Yes, this gets baked in at compilation time, so we'll need a completely different approach here.

Reply:
It's ok that it. requires restarting the repl. In the current approach we still need to do that. This is a sufficiently good approach to not need to change the source code every time.

Reply:
Currently it requires not just restarting the REPL but the whole manual re-precompliation thing we were trying to avoid in the first place. Confident I can figure out an approach that won't require re-precompilation but I don't have it yet.